src/pretalx/orga/templates/orga/mails/outbox_list.html

```diff
@@ -102,6 +103,7 @@ <h2>
                                 {% elif mail.template == request.event.question_template %}
                                     <span class="badge color-info">{% translate "Unanswered questions reminder" %}</span>
                                 {% endif %}
+                                {% html_signal "pretalx.mail.signals.html_after_mail_badge" sender=request.event request=request mail=mail %}
```

Comment:
I worry about the performance hit for calling a signal on every row of a potentially 100+ row table, especially as plugins can't add fields to be prefetched/preselected, which sounds like a recipe for n+1 queries.

This kind of request makes me think that maybe we should move to using `django-tables2` (or a ripoff) regardless of the unfortunate performance penalty detailed [here](https://github.com/pretalx/pretalx/pull/1589), as then we could give plugins the option to register proper table columns instead.